test/e2e/proxy.go

```diff
@@ -224,6 +224,12 @@ func proxyContext(version string) {
 					body, status, d, err := doProxy(f, path)
 					if err != nil {
 						recordError(fmt.Sprintf("%v: path %v gave error: %v", i, path, err))
+						body, err := f.Client.Pods(f.Namespace.Name).GetLogs(pods[0].Name, &api.PodLogOptions{}).Do().Raw()
+						if err != nil {
+							framework.Logf("Error getting logs for pod %s: %v", pods[0].Name, err)
```

Comment:
Will move the error output down outside of the loop. `framework.Logf` should be correct.